cumulusci/tasks/metadeploy.py

```diff
@@ -219,6 +282,8 @@ def _find_or_create_version(self, product):
         return version
 
     def _find_or_create_plan_template(self, product, plan_name, plan_config):
+        # TODO: Add preflight_message, post_install_message, and error_message here
+        # once the MetaDeploy API supports them.
```

Comment:
@codexmarcel they are added in the MetaDeploy PR you looked at: https://github.com/SFDO-Tooling/MetaDeploy/pull/977/files#diff-7b9b524e363875f277fe63e6a912f5c2R73 (This is a Django Rest Framework serializer used by the view that serves the "admin API" that CumulusCI interacts with. These fields were not included by the default serializer because we have a special field type to handle Markdown and it's not fully integrated with DRF.)

I should be able to make this adjustment here once I deploy the MetaDeploy changes.